.circleci/src/jobs/install_packages_to_server.yml

```diff
@@ -0,0 +1,31 @@
+docker:
+  - image: cimg/base:2022.09
```

Comment:
Whouldn't it make more sense to use a debian package here too? Not really important, so fine for me to keep an ubuntu image.

Reply:
i think we can address this in the unified images issue